### DISCUSSION TITLE
In our sentry logs we get Failed to fetch over and over

### DISCUSSION THREAD

**Discussion by northkode** [getsentry/support#47] [Support]
I am trying to figure this error, and understand if it really is an error, or just sentry getting confused. 

We have had no reports from customers of issues, however we get hundreds of "Failed to fetch" errors. 

There is no details in these errors about the XHR request that was tried etc. its impossible to debug cause we cant actually get these errors when using out app. 

Is there a solution to this or a known way to debug these errors?

**Comment by mitsuhiko** [comment]
In which logs do you see these? Is this a client log or on the Sentry server?
